src/analysis/normed/group/basic.lean

```diff
@@ -1020,6 +1020,14 @@ begin
   exact (tendsto_at_top_of_eventually_const this).cauchy_seq.add hv
 end
 
+lemma tendsto_sub_nhds_zero_iff
```

Comment:
Shouldn't we prove this for any topological group (multiplicative or additive) rather than just normed?